src/main/java/uk/ac/ebi/ena/webin/cli/context/reads/ReadsManifestReader.java

```diff
@@ -121,7 +119,7 @@ public ReadsManifestReader(
                        .meta().optional().name(Field.LIBRARY_NAME).desc(Description.LIBRARY_NAME).and()
                        .meta().optional().name(Field.INSERT_SIZE).desc(Description.INSERT_SIZE).and()
 
-                       .file().optional(2).name(Field.FASTQ).desc(Description.FASTQ).processor(getFastqProcessors())
+                       .file().optional(999).name(Field.FASTQ).desc(Description.FASTQ).processor(getFastqProcessors())
```

Comment:
Thought about this more this morning and I suggest that we cap the number Fastq files to 10 to start with.